src/main/java/se/ayoy/maven/plugins/licenseverifier/LicenseInfo/LicenseInfoFile.java

```diff
@@ -43,10 +47,21 @@ public LicenseInfoFile(String filePathString, Log log)
                         + filePathString);
 
         File file = new File(filePathString);
+        InputStream inputStream;
```

Comment:
Possibly a bit pedantic, but this could be more readable as a getInputStream()-method